src/components/results/securityanalysis/security-analysis-result-utils.ts

```diff
@@ -382,23 +486,35 @@ export const useFetchFiltersEnums = (isEmptyResult: boolean = true) => {
     const [error, setError] = useState(false);
     const [result, setResult] = useState({
         computationsStatus: null,
+        limitTypes: null,
+        branchSides: null,
     });
 
     useEffect(() => {
         const fetchAllData = () => {
             const promises = [
                 // We can add another fetch for other enums
                 fetchSecurityAnalysisAvailableComputationStatus(),
+                fetchSecurityAnalysisAvailableLimitTypes(),
+                fetchSecurityAnalysisAvailableBranchSides(),
```

Comment:
I haven't tested it but I think we might fetch all of these data for N and NmK tabs each time we switch between them

Reply:
Fixed